### PR TITLE
feat(ChangelogDialog): change style, add link, events

### PR DESCRIPTION
**Pull request by Feverqwe**

### CODE REVIEW

**Review by DaffPunks — APPROVED** [review]

**Review by korvin89 — CHANGES_REQUESTED** [review]
Look like major updates? cc @amje

**Comment by Feverqwe** [comment]
> Look like major updates?

No, we don't use it yet. And just add one optional prop

**Comment by korvin89** [comment]
> > Look like major updates?
> 
> No, we don't use it yet. And just add one optional prop

You don't use, but others could

**Comment by amje** [comment]
@Feverqwe Removing prop is breaking change. Mark as deprecated and remove in the next major release

**Comment by Feverqwe** [comment]
> You don't use, but others could

Ok, I remove breaking changes, and just add link feature and change style

File: src/components/ChangelogDialog/components/Item/Item.scss [hunk 1]
```diff
@@ -17,17 +17,18 @@ $metaWidth: 124px;
...
         flex: 1;
-        margin-left: 16px;
+        margin-left: 20px;
     }
 
     &__title {
         margin: 0;
-        @include mixins.text-subheader-3();
+        @include mixins.text-subheader-2();
+        font-weight: 500;
```
Comment (amje) [inline src/components/ChangelogDialog/components/Item/Item.scss:31]: mixin already has font-weight. Besides it uses variable for that, not a fixed value

**Review by amje — CHANGES_REQUESTED** [review]

File: src/components/ChangelogDialog/components/Item/Item.scss (hunk 1 above)
Comment (Feverqwe) [reply src/components/ChangelogDialog/components/Item/Item.scss]: fixed, thx!

**Review by amje — APPROVED** [review]

**Comment by Feverqwe** [comment]
@amje I'm add events, rm metrics

File: src/components/ChangelogDialog/ChangelogDialog.tsx [hunk 2]
```diff
@@ -38,12 +40,19 @@ export function ChangelogDialog({
...
+    onOpen,
     onStoryClick,
+    onLinkClick,
 }: ChangelogDialogProps) {
     const idRef = React.useRef<number>();
     idRef.current = idRef.current || getNextId();
     const dialogCaptionId = `changelog-dialog-title-${idRef.current}`;
 
+    useEffect(() => {
+        if (!open) return;
```
Comment (ogonkov) [inline src/components/ChangelogDialog/ChangelogDialog.tsx:52]: What the reason to add callback, if you pass `open` from outside anyway?

File: src/components/ChangelogDialog/ChangelogDialog.tsx [hunk 3]
```diff
@@ -21,7 +21,9 @@ export interface ChangelogDialogProps {
     items: ChangelogItem[];
     disableBodyScrollLock?: boolean;
     disableOutsideClick?: boolean;
+    onOpen: () => void;
```
Comment (amje) [inline src/components/ChangelogDialog/ChangelogDialog.tsx:24]: Why it is required?

File: src/components/ChangelogDialog/ChangelogDialog.tsx (hunk 2 above)
Comment (amje) [reply src/components/ChangelogDialog/ChangelogDialog.tsx:52]: Looks like there is no need to keep this code inside the component

**Review by amje — CHANGES_REQUESTED** [review]

File: src/components/ChangelogDialog/ChangelogDialog.tsx (hunk 2 above)
Comment (Feverqwe) [reply src/components/ChangelogDialog/ChangelogDialog.tsx]: Yes, you are right.

File: src/components/ChangelogDialog/ChangelogDialog.tsx (hunk 3 above)
Comment (Feverqwe) [reply src/components/ChangelogDialog/ChangelogDialog.tsx]: Remove, will do in parent component

File: src/components/ChangelogDialog/components/Item/Item.scss [hunk 4]
```diff
@@ -17,17 +17,17 @@ $metaWidth: 124px;
     }
 
     &__label-new {
-        margin-top: 4px;
+        margin-top: 8px;
```
Comment (ogonkov) [inline src/components/ChangelogDialog/components/Item/Item.scss:20]: I guess we should use `--g-spacing-2`

File: src/components/ChangelogDialog/components/Item/Item.scss [hunk 5]
```diff
@@ -17,17 +17,17 @@ $metaWidth: 124px;
...
     }
 
     &__content {
         flex: 1;
-        margin-left: 16px;
+        margin-left: 20px;
```
Comment (ogonkov) [inline src/components/ChangelogDialog/components/Item/Item.scss:25]: I guess we should use `--g-spacing-5`

Comment (Feverqwe) [reply src/components/ChangelogDialog/components/Item/Item.scss]: Replace this and other places too.

**Review by amje — APPROVED** [review]

**Pull request merged**